src/api/getPosts.ts

```diff
@@ -0,0 +1,13 @@
+import { toJson } from '../heplers/toJson'
+import type { Post } from '../types/models'
+
+export const getPosts = async (): Promise<Array<Post>> => {
+  const response = await fetch('http://localhost:3002/api/posts', {
```

Comment:
```suggestion
  const response = await fetch(`${import.meta.env.VITE_BACKEND_URL}/api/posts`, {
```

Please check: https://vite.dev/guide/env-and-mode